webapp/pages/notifications/index.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <ds-card space="small">
+    <ds-flex>
+      <ds-flex-item :width="{ lg: '85%' }">
+        <ds-heading tag="h3">{{ $t('notifications.title') }}</ds-heading>
+      </ds-flex-item>
+      <ds-flex-item class="sorting-dropdown">
```

Comment:
Here the styleguide messes things up again... If you don't give the `ds-flex-item` a `width` it will have a `flex-grow: 1` inline style so it will take up the whole remaining width of the flexbox, making it impossible to align it right.

As a workaround I would suggest:
1. give the `ds-flex-item` a `width="100px"`
2. remove the `sorting-dropdown` class
3. give the `ds-flex` a class with `justify-content: space-between;`

Reply:
great trick!